Build/xTVS.ruleset

```diff
@@ -96,5 +96,117 @@
     <Rule Id="CA2226" Action="Warning" />
     <Rule Id="CA2227" Action="Warning" />
     <Rule Id="CA2239" Action="Warning" />
+
+    <Rule Id="CA1304" Action="Error" /> <!-- specify CultureInfo -->
+    <Rule Id="CA1305" Action="None" />  <!-- provide IFormatProvider -->
```

Comment:
Can you also switch `CA1063` to `None`?

Reply:
Sure. It's not in the required list for releasing.